packages/components/src/Menu/Menu.module.css

```diff
@@ -65,24 +65,26 @@
 
 .action {
   display: flex;
-  gap: var(--menu-space);
   width: 100%;
   padding: var(--menu-space);
   border: none;
   border-radius: var(--radius-base);
+  outline: transparent;
```

Comment:
Beacuse we're using box-shadow on focus, this sets outline to transparent so that high-contrast modes still work.